tests/unit/test_package_managers.py

```diff
@@ -148,6 +150,8 @@ def assert_output(
     (CONTAINER_FEDORA_39, PACKAGE_MANAGER_YUM),
 
     # CentOS Stream
+    (CONTAINER_CENTOS_STREAM_10, PACKAGE_MANAGER_DNF),
+    (CONTAINER_CENTOS_STREAM_10, PACKAGE_MANAGER_YUM),
```

Comment:
Do we really need to exercise `yum` for `centos-stream-10`?

Reply:
Good point, dropped in 187ef946